<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resource Builder</string>
  </property>
  <layout class="QVBoxLayout" name="uiMainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="uiSplitterLayout">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="uiLeftLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QListWidget" name="uiRuleList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="uiButtonsLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="uiNewFileButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Create new file</string>
           </property>
           <property name="icon">
            <iconset resource="res/resources.qrc">
             <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="uiOpenFileButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Open existing file</string>
           </property>
           <property name="icon">
            <iconset resource="res/resources.qrc">
             <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="uiSaveFileButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Save current file</string>
           </property>
           <property name="icon">
            <iconset resource="res/resources.qrc">
             <normaloff>:/icons/media-floppy.png</normaloff>:/icons/media-floppy.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="uiButtonSpacer2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="uiAddRuleButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Add rule</string>
           </property>
           <property name="icon">
            <iconset resource="res/resources.qrc">
             <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="uiRemoveRuleButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Remove selected rule</string>
           </property>
           <property name="icon">
            <iconset resource="res/resources.qrc">
             <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="uiButtonSpacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="uiDraftBuildButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Perform a fast (draft) build</string>
           </property>
           <property name="icon">
            <iconset resource="res/resources.qrc">
             <normaloff>:/icons/fatcow_modified/hammer_lightning.png</normaloff>:/icons/fatcow_modified/hammer_lightning.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="uiFinalBuildButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Perform a slow (final) build</string>
           </property>
           <property name="icon">
            <iconset resource="res/resources.qrc">
             <normaloff>:/icons/fatcow_modified/hammer_ok.png</normaloff>:/icons/fatcow_modified/hammer_ok.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="uiCleanButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Delete all generated files</string>
           </property>
           <property name="icon">
            <iconset resource="res/resources.qrc">
             <normaloff>:/icons/edit-clear.png</normaloff>:/icons/edit-clear.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="uiContentsArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>100</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="uiContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>606</width>
         <height>702</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uiRuleList</tabstop>
  <tabstop>uiNewFileButton</tabstop>
  <tabstop>uiOpenFileButton</tabstop>
  <tabstop>uiSaveFileButton</tabstop>
  <tabstop>uiAddRuleButton</tabstop>
  <tabstop>uiRemoveRuleButton</tabstop>
  <tabstop>uiDraftBuildButton</tabstop>
  <tabstop>uiFinalBuildButton</tabstop>
  <tabstop>uiCleanButton</tabstop>
  <tabstop>uiContentsArea</tabstop>
 </tabstops>
 <resources>
  <include location="res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
